GTG/gtk/browser/main_window.py

```diff
@@ -1041,6 +1046,11 @@ def update_start_to_next_day(self, day_number):
         for task in self.get_pane().get_selection():
             task.date_start = next_day
 
+        # Changing the start date of a task may take it out of the current pane
```

Comment:
Note that we're again running into the fact that the same functionality (updating the start date of a task) is implemented in two different methods (`update_start_date()` and `update_start_to_next_day()`).

I'd like to have one call the other instead, but that's not really related to the current change so I didn't include it here.